roles/ezproxy/tasks/main.yml

```diff
@@ -174,47 +142,53 @@
   when:
     - running_on_server
 
-- name: Ezproxy | install ezproxy startup script (requires root)
-  ansible.builtin.command: /var/local/ezproxy/ezproxy -si
-  become: true
-  become_user: 'root'
-  changed_when: false
-  when: running_on_server
-
-
 - name: Ezproxy | add shib config user
   ansible.builtin.template:
     src: "shibuser.txt.j2"
     dest: /var/local/ezproxy/shibuser.txt
-    mode: 0644
+    mode: "0644"
     owner: root
     group: root
 
+- name: Ezproxy | add shib config user
```

Comment:
I don't really understand what this task does from the name. Looking at the template, it looks to be adding IPs or IP ranges, not users . . . was this copy pasta?